tools/java/org.hl7.fhir.tools.core/src/org/hl7/fhir/tools/publisher/Publisher.java

```diff
@@ -336,7 +337,7 @@ public String getPath() {
   private String singlePage;
 
   private Map<String, Example> processingList = new HashMap<String, Example>();
-  

```

Comment:
Ha.  Yeah.  I really dislike trailing spaces, inconsistent indents, etc.  It drives me crazy.  So usually I appreciate it when Atom does that for my own code-- but it's kind of annoying when it applies it to an entire file of which you've only written two lines.